compiler/cbmc/src/goto_program/symbol.rs

```diff
@@ -289,12 +323,12 @@ impl Symbol {
 impl Symbol {
     /// Makes a formal function parameter from a symbol.
     pub fn to_function_parameter(&self) -> Parameter {
-        self.typ.clone().as_parameter(Some(self.name.to_string()), self.base_name.clone())
+        self.typ.clone().as_parameter(Some(self.name), self.base_name.clone().map(|n| n.into()))
```

Comment:
if we're doing a map, do we still need the clone?

Reply:
nope!